guides/common/modules/con_provisioning-cloud-instances-on-microsoft-azure-resource-manager.adoc

```diff
@@ -3,13 +3,15 @@
 [id="Provisioning_Cloud_Instances_on_Microsoft_Azure_Resource_Manager_{context}"]
 = Provisioning cloud instances on Microsoft Azure Resource Manager
```

Comment:
out of scope unless you really want to: I suggest that we shorten this to "Provisioning cloud instances on Microsoft Azure" (same for anchor + file name).

You can leave this thread open so we can create a follow-up issue after merging your PR.

Reply:
https://github.com/theforeman/foreman-documentation/issues/4535